frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/ERDContent.tsx

```diff
@@ -65,47 +65,66 @@ export const ERDContentInner: FC<Props> = ({
   const {
     state: { loading },
   } = useERDContentContext()
+  const [clickedNodeId, setClickedNodeId] = useState<string | null>(null)
```

Comment:
It is painful to have to create a separate state from the `active` state managed by `useUserEditingStore()`, but it seems that the logic has to be unravelled to solve the problem. I'd like to try another PR.

Reply:
You're absolutely right. I wanted to write the code more simply, but I couldn't. In the end, the code became more complex🥲.Thank you for thinking about it🙏.